sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuOrcScan.scala

```diff
@@ -2067,7 +2067,9 @@ class MultiFileCloudOrcPartitionReader(
               while (blockChunkIter.hasNext) {
                 val blocksToRead = populateCurrentBlockChunk(blockChunkIter, maxReadBatchSizeRows,
                   maxReadBatchSizeBytes)
-                val (hostBuf, bufSize) = readPartFile(ctx, blocksToRead)
+                val (hostBuf, bufSize) = metrics(BUFFER_TIME).ns {
```

Comment:
hmm, but BUFFER_TIME is being set, by way of the `setMetrics` method on line 2096. The problem with changing BUFFER_TIME in this part of the code, is that we are in one of the IO threads and BUFFER_TIME was supposed to be relative to the task time (it's the amount of time the task waited for the buffering to finish, not the amount of time all threads took in aggregate to buffer).

Reply:
yes, you're right 👍 

i'll need to rethink this.  after investigating more, it looks like bufferTime isn't logged in the MultiFileOrcPartitionReader (not Cloud one), so that is likely where it needs to be added.

Reply:
Do we need to want to be consistent with what we're doing in [Parquet scan path](https://github.com/NVIDIA/spark-rapids/blob/branch-23.10/sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuParquetScan.scala#L2832-L2834)?